cmd/kind/version/version.go

```diff
@@ -35,7 +35,8 @@ func Version() string {
 		// if commit was set, add the + <build>
 		// we only do this for pre-release versions
 		if GitCommit != "" {
-			v += "+" + GitCommit
+			// NOTE: use 14 character short hash, like Kubernetes
+			v += "+" + truncate(GitCommit, 14)
```

Comment:
Git says 12 characters is suficcient for the linux kernel, but kubernetes uses 14 and this still seems pretty short
https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection
https://github.com/kubernetes/kubernetes/blob/6bcfd75dd662625b2f92c41cc19fb0fd1cc5aa32/hack/lib/version.sh#L69

```
$ kind version
kind v0.6.0-alpha+cb1e6727f4d507 go1.13.3 linux/amd64
```
/shrug
we can always revisit and truncate more or less, this is trivial to change now.